lib/susedistribution.pm

```diff
@@ -567,6 +567,15 @@ sub activate_console {
     elsif ($type eq 'virtio-terminal') {
         serial_terminal::login($user, $self->{serial_term_prompt});
     }
+    elsif ($console eq 'novalink-ssh') {
+        check_screen "password-prompt-novalink", 5;
+        type_string get_required_var('NOVALINK_PASSWORD'), secret => 1;
+        send_key('ret');
+        my $user = get_var('NOVALINK_USERNAME', 'root');
+        #assert_screen(["text-logged-in-$user", "text-login"], 60);
+        sleep 3;
```

Comment:
why?

Reply:
I guess because our needles don't match the ubuntu based novalink login prompt.